client/sections.js

```diff
@@ -22,21 +22,20 @@ const extensionSections = extensions.map( extension => {
 	}
 } );
 
-if ( config.isEnabled( 'devdocs' ) ) {
-	sections.push( {
-		name: 'devdocs',
-		paths: [ '/devdocs' ],
-		module: 'devdocs',
-		secondary: true,
-		enableLoggedOut: true
-	} );
+sections.push( {
```

Comment:
@mtias: I'm not sure how generating /not-generating bundles can affect extensibility.
Whether or not the client requests the bundle is up to it.  For example, theres no reason the `devdocs` bundle can't exist on the server for `NODE_ENV=production`, as long as the client doesn't register the routes via `page`